.github/actions/setup-minikube/action.yaml

```diff
@@ -3,6 +3,13 @@ description: 'Installs minikube Kubernetes cluster'
 runs:
   using: "composite"
   steps:
+  - name: Configure system
+    shell: bash
+    run: |
+      set -euExo pipefail
+
+      echo 'fs.aio-max-nr = 10485760' | sudo tee /etc/sysctl.d/90-scylla.conf >/dev/null
```

Comment:
Was this number printed in scylla logs? According to https://github.com/scylladb/scylla/pull/8228/files this should be 87588. I don't mind higher number, just wondering where it's coming from.

Reply:
This number accounts for **multiple** scylla instances running on the same node. It's just an arbitrarily high number, hopefully high enough. (Practically it's just a 10x what I had on my dev machine.)